plugin-server/tests/worker/ingestion/person-state.test.ts

```diff
@@ -65,606 +68,1719 @@ describe('PersonState.update()', () => {
-                id: expect.any(Number),
-                uuid: uuid2.toString(),
-                properties: { a: 1, b: 3, c: 4, d: 6, e: 7, f: 9 },
-                created_at: timestamp,
-                is_identified: true,
-                version: 1,
-            })
-        )
-        const distinctIds = await hub.db.fetchDistinctIdValues(persons[0])
-        expect(distinctIds).toEqual(expect.arrayContaining(['new-user', 'old-user']))
+    describe('on $identify event', () => {
+        it('creates person', async () => {
+            const personContainer = await personState({
+                event: '$identify',
+                distinct_id: 'new-user',
+                properties: {
+                    $set: { foo: 'bar' },
+                },
+            }).update()
 
-        const clickhousePersons = await delayUntilEventIngested(() => fetchPersonsRows(), 2)
-        expect(clickhousePersons.length).toEqual(2)
-        expect(clickhousePersons).toEqual(
-            expect.arrayContaining([
+            // verify Postgres persons
+            const persons = await hub.db.fetchPersons()
+            expect(persons.length).toEqual(1)
+            expect(persons[0]).toEqual(
                 expect.objectContaining({
-                    id: uuid2.toString(),
-                    properties: JSON.stringify({ a: 1, b: 3, c: 4, d: 6, e: 7, f: 9 }),
-                    is_deleted: 0,
-                    created_at: '2020-01-01 12:00:05.000',
-                    version: 1,
-                }),
+                    id: expect.any(Number),
+                    uuid: uuid.toString(),
+                    properties: { foo: 'bar' },
+                    created_at: timestamp,
+                    version: 0,
+                    is_identified: false, // is_identified is not tied to $identify calls but used for safeguarding bad merges
+                })
+            )
+
+            // verify Postgres distinct_ids
+            const distinctIds = await hub.db.fetchDistinctIdValues(persons[0])
+            expect(distinctIds).toEqual(expect.arrayContaining(['new-user']))
+
+            // verify ClickHouse persons
+            const clickhousePersons = await delayUntilEventIngested(fetchPersonsRows)
+            expect(clickhousePersons.length).toEqual(1)
+            expect(clickhousePersons[0]).toEqual(
                 expect.objectContaining({
                     id: uuid.toString(),
-                    is_deleted: 1,
-                    version: 100,
-                }),
-            ])
-        )
+                    properties: JSON.stringify({ foo: 'bar' }),
+                    created_at: timestampch,
+                    version: 0,
+                    is_identified: 0,
+                })
+            )
 
-        expect(hub.personManager.isNewPerson).toHaveBeenCalledTimes(0)
-        expect(hub.db.fetchPerson).toHaveBeenCalledTimes(2)
-    })
+            // verify ClickHouse distinct_ids
```

Comment:
I'm not a fan of the pattern of verifying both ClickHouse and postgres at once since this class does not know the difference.

If there are issues with keeping ClickHouse and postgres in sync, these should be tested for in the lower level primitives not here. As-is this is just adding noise to the tests and making them harder to write/maintain.

Reply:
Will remove from all create and update describe paths. 
identify/alias - consider removing too (only in merge path)